src/index.html

```diff
@@ -17,6 +17,90 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="container">
+        <a
```

Comment:
Don't use spaces in `<a>` tag's `href` property. If this is supposed to be a link to the top of the page, use `href="#top"` or an appropriate id.